x-pack/platform/plugins/shared/fleet/public/applications/integrations/sections/epm/screens/detail/overview/prerelease_callout.tsx

```diff
@@ -0,0 +1,129 @@
+
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import type { PackageInfo } from '../../../../../types';
+
+import { useLink, useStartServices } from '../../../../../hooks';
+import { isPackagePrerelease } from '../../../../../../../../common/services';
+import {
+  OTEL_INPUTS_MINIMUM_VERSION,
+  packageInfoHasOtelInputs,
+} from '../../../../../../../../common/services/otelcol_helpers';
+import { ExperimentalFeaturesService } from '../../../../../services';
+
+export const PrereleaseCallout: React.FC<{
+  packageInfo: PackageInfo;
+  latestGAVersion?: string;
+}> = ({ packageInfo, latestGAVersion }) => {
+  const { getHref } = useLink();
+  const { enableOtelIntegrations } = ExperimentalFeaturesService.get();
+  const { name, title } = packageInfo;
+  const overviewPathLatestGA = getHref('integration_details_overview', {
+    pkgkey: `${name}-${latestGAVersion}`,
+  });
+  const isPrerelease = isPackagePrerelease(packageInfo.version);
+  if (!isPrerelease) return null;
+
+  if (enableOtelIntegrations && packageInfoHasOtelInputs(packageInfo)) {
+    return <OtelPackageCallout packageInfo={packageInfo} />;
+  }
+  return (
+    <>
+      <EuiCallOut
+        announceOnMount
+        data-test-subj="prereleaseCallout"
+        title={i18n.translate('xpack.fleet.epm.prereleaseWarningCalloutTitle', {
+          defaultMessage: 'This is a pre-release version of {packageTitle} integration.',
+          values: {
+            packageTitle: title,
+          },
+        })}
+        iconType="info"
+        color="warning"
+      >
+        {latestGAVersion && (
+          <p>
+            <EuiButton href={overviewPathLatestGA} color="warning" data-test-subj="switchToGABtn">
+              <FormattedMessage
+                id="xpack.fleet.epm.prereleaseWarningCalloutSwitchToGAButton"
+                defaultMessage="Switch to latest GA version"
+              />
+            </EuiButton>
+          </p>
+        )}
+      </EuiCallOut>
+    </>
+  );
+};
+
+export const OtelPackageCallout: React.FC<{
+  packageInfo: PackageInfo;
+}> = ({ packageInfo }) => {
+  const { docLinks } = useStartServices();
+  const packageTitle = packageInfo.title;
+
+  return (
+    <>
+      <EuiCallOut
+        announceOnMount
+        data-test-subj="prereleaseCallout"
+        title={i18n.translate('xpack.fleet.epm.otelPackageWarningTitle', {
+          defaultMessage: 'This is a pre-release version of {packageTitle} integration.',
+          values: {
+            packageTitle,
+          },
+        })}
+        iconType="warning"
+        color="warning"
+      >
+        <p>
+          <FormattedMessage
+            id="xpack.fleet.epm.otelPackageWarningMessage"
+            defaultMessage="The {packageTitle} integration collects {OTelExternalLink} data adhering to {semanticConventionsLink}, and is available in technical preview. Elastic Agents must be on version {minVersion} or higher to collect OTel data. For more information, see the {fleetUserGuide}."
```

Comment:
(we avoid using “see” for accessibility reasons)

```suggestion
            defaultMessage="The {packageTitle} integration collects {OTelExternalLink} data adhering to {semanticConventionsLink}, and is available in technical preview. To collect OTel data, Elastic Agents must be on version {minVersion} or later. For more information, refer to the {fleetUserGuide}."
```

Reply:
Oh ok, we have it in a lot of places. I guess we'll have to update it everywhere then